include/fiction/technology/sidb_lattice.hpp

```diff
@@ -64,7 +64,7 @@ class sidb_lattice<LatticeOrientation, Lyt, false> : public Lyt
      * @param ar Highest possible position in the layout.
      * @param name Layout name.
      */
-    explicit sidb_lattice(const aspect_ratio<Lyt>& ar, const std::string& name = "") : Lyt(ar, name)
+    explicit sidb_lattice(const typename Lyt::aspect_ratio& ar, const std::string& name = "") : Lyt(ar, name)
```

Comment:
Since you defined the trait, calls like these can be written as 
```suggestion
    explicit sidb_lattice(const aspect_ratio_t<Lyt>& ar, const std::string& name = "") : Lyt(ar, name)
```